src/node_url.cc

```diff
@@ -829,6 +829,16 @@ static url_host_type ParseOpaqueHost(url_host* host,
   return type;
 }
 
+static inline bool IsAllASCII(std::string* input) {
```

Comment:
@mscdex : That function is inside anon namespace and is static. Do you off hand know any place where we can refactor it in some utility?.